src/translations/en.json

```
@@ -5705,7 +5705,15 @@
           "visualization": {
             "header": "Network visualization",
             "caption": "Visualization",
-            "refresh_topology": "Refresh topology"
+            "refresh_topology": "Refresh topology",
+            "device": "Device",
+            "device_type": "Device type",
+            "device_not_in_db": "Device is not in <i>'zigbee.db'</i>",
```

```suggestion
            "device_not_in_db": "Device is not in {zigbeeDb}",
```

This leads to a translation error. You would need to add the `i` in the variable and add an additional translation for zigbee.db. Or you just remove the `i` :slightly_smiling_face: